src/mame/drivers/route16.cpp

```diff
@@ -1107,4 +1107,5 @@ GAME( 1980, spacecho, speakres, spacecho, spacecho, route16_state, empty_init,
 GAME( 1980, spacecho2,speakres, spacecho, spacecho, route16_state, empty_init,    ROT270, "bootleg (Gayton Games)",          "Space Echo (set 2)", MACHINE_SUPPORTS_SAVE )
 GAME( 1980, speakhlp, speakres, spacecho, spacecho, route16_state, empty_init,    ROT270, "bootleg",                         "Speak & Help", MACHINE_SUPPORTS_SAVE | MACHINE_IMPERFECT_SOUND )
 
-GAME( 1981, ttmahjng, 0,        ttmahjng, ttmahjng, route16_state, empty_init,    ROT0,   "Taito", "T.T Mahjong", MACHINE_SUPPORTS_SAVE )
+GAME( 1981, ttmahjng, 0,        ttmahjng, ttmahjng, route16_state, empty_init,    ROT0,   "Taito",            "T.T Mahjong", MACHINE_SUPPORTS_SAVE )
+GAME( 1981, jongpute, 0,        ttmahjng, ttmahjng, route16_state, empty_init,    ROT0,   "Alpha Denshi Co.", "Jongputer",   MACHINE_SUPPORTS_SAVE | MACHINE_IMPERFECT_SOUND )  // sampling voice is not emulated
```

Comment:
Yep.
So, since the jongpute is the original and ttmahjng is the one that should be marked as a clone, do you think I should rename existing ttmahjng_* methods and members to jongpute_*?

Reply:
The second commit fixes it and does the renaming.